### PR TITLE
ignition-fuel_tools-4: Fix checksums and dir name

### PR DESCRIPTION
**Pull request by alexdewar**
This is a similar issue to the other ones we've been having with ``ignition-*`` packages.

### CODE REVIEW

File: ignition-fuel_tools-4/PKGBUILD [hunk 1]
```diff
@@ -14,9 +14,9 @@ makedepends=('ignition-cmake>=2')
 optdepends=('ruby')
 conflicts=()
 source=("${pkgname}-${pkgver}.tar.gz::https://github.com/ignitionrobotics/ign-fuel-tools/archive/ignition-fuel-tools4_${pkgver}.tar.gz")
```
Comment (acxz) [inline ignition-fuel_tools-4/PKGBUILD:16]: This should also be changed to `gazebosim/gz-fuel-tools` i think
and the `url` field.

**Review by acxz — CHANGES_REQUESTED** [review]

File: ignition-fuel_tools-4/PKGBUILD (hunk 1 above)
Comment (alexdewar) [reply ignition-fuel_tools-4/PKGBUILD]: Done!

Comment (acxz) [reply ignition-fuel_tools-4/PKGBUILD]: The `url` field still needs to be changed

Comment (alexdewar) [reply ignition-fuel_tools-4/PKGBUILD]: So it does.

**Review by acxz — APPROVED** [review]

**Pull request merged**